package.json

```diff
@@ -5,17 +5,17 @@
   "main": "index.js",
   "dependencies": {
     "execspawn": "^1.0.1",
-    "minimist": "^1.2.0",
-    "mkdirp": "^0.5.1",
-    "node-abi": "^2.0.0",
-    "npm-run-path": "^3.1.0",
+    "mkdirp": "^1.0.4",
+    "node-abi": "^2.19.1",
+    "npm-run-path": "^4.0.1",
```

Comment:
What's the reason for the major bump in `npm-run-path`?

Reply:
Nothing affecting us:
https://github.com/sindresorhus/npm-run-path/releases/tag/v4.0.0

Reply:
I'm not sure, specifically about https://github.com/sindresorhus/npm-run-path/commit/15fe40fe453e8ed73292c35a9067f290857453df. If the directory that contains `node` also contains `node-gyp` (which can happen with nvm IIRC) then a globally-installed `node-gyp` would take precedence over a locally-installed `node-gyp`.

Reply:
I downgraded the package. https://github.com/prebuild/prebuildify/pull/45/commits/89708ea4fed14090e5a746239b5b380fc3349d4c

I will let you investigate the breakage potential.